src/lib/chat/chat-message.test.ts

```diff
@@ -274,6 +274,35 @@ describe(adminMessageText, () => {
       expect(adminText).toEqual('You started the conversation');
     });
   });
+
+  describe(AdminMessageType.MEMBER_LEFT_CONVERSATION, () => {
+    it('returns default message if creator not found', () => {
+      const state = getState('current-user', {});
+      const adminText = adminMessageText(
+        {
+          message: 'some message',
+          isAdmin: true,
+          admin: { type: AdminMessageType.MEMBER_LEFT_CONVERSATION, creatorId: 'unknown-user-id' },
+        } as Message,
+        state
+      );
+
+      expect(adminText).toEqual('some message');
+    });
+
+    it('translates message if current user was not the creator', () => {
```

Comment:
I'll also replace the content of `it` here with `translates message if creator is found` as we do not check for current user for this admin message type.